src/soundio/sounddeviceportaudio.cpp

```diff
@@ -360,12 +364,14 @@ SoundDeviceStatus SoundDevicePortAudio::open(bool isClkRefDevice, int syncBuffer
     }
 #endif
 
-    // Start stream
+    // Set the finished callback, used when stopping the stream
```

Comment:
```suggestion
    // Set the finished callback, informs us via m_bFinishedCV when Pa_IsStreamStopped() becomes true
    // and we can safely call Pa_CloseStream() 
```

Reply:
Did you confirm by looking into the portaudio code that Pa_IsStreamStopped() coincides with the finished callback? (It's what I  understand from the documentation, but confirming this would be good, if we add this in the comments)